docs/getting_started/requirements_and_system_configuration.md

```diff
@@ -37,18 +37,21 @@ Supported setups are those we perform automated testing on. For security and per
 
 -   OS: Linux
 -   Web Servers:
-    -   Apache 2.4, with required modules `mod_php`, `mod_rewrite`, `mod_env` and recommended: `mod_setenvif`, `mod_expires`
-    -   Nginx 1.10, 1.12
+    -   Apache 2.4, with required modules `mod_rewrite`, `mod_env` and recommended: `mod_setenvif`, `mod_expires`
+        - event MPM is recommended, if you for some reason need to use _prefork_ you'll also need `mod_php` module
+    -   Nginx 1.10, 1.12, 1.14
 -   DBMS
     -   MySQL 5.5.3 and higher, 5.6\*, 5.7\*
     -   MariaDB 5.5, 10.0, 10.1, 10.2\*
 -   PHP
     -   7.1
+    -   7.2
 
 - Cluster
-    - Redis or Memcached
-    - Solr or SQL *(but does not provide same featureset or performance as Solr)*
+    - Redis or Memcached *(Not recommended as it has large performance issues with cache tagging by Symfony Cache component)*
```

Comment:
If I am right only `Memcached` has performances issues, right?
Maybe we should remove it.

Reply:
We can, we can rather mention it among experimental setups until the performance issue is solved in Symfony.

Reply:
fixed in 5f3a10f

Reply:
And some more in 4afed93

Reply:
Thanks. Definitely better.